packages/components/src/docs/General/11-FAQ.mdx

```diff
@@ -151,34 +135,23 @@ import 'https://solid-design-system.fe.union-investment.de/x.x.x/components/es/a
           Which browsers are supported?
         </h4>
         <span class="sd-prose">
-          Currently we provide automatic VRT in Chrome and manually test our components with MS Edge, Safari and Firefox. A
-          more detailed browser matrix is in progress and will be added here soon.
+          Currently the we provide auomatic VRT in Chrome and manually test our components with MS Edge, Safari and Firefox. A more detailed browser matrix is in progress and will be added here soon.
         </span>
       </sd-accordion>
-
       <sd-accordion>
         <h4 class="text-base" slot="summary">
           Does the design system support TypeScript and work with React, Vue, and Angular?
         </h4>
         <span class="sd-prose">
-          The Solid Design System is designed to be framework-agnostic, meaning it can be integrated with various frameworks,
-          including React, Vue, Angular and <a target="_blank" href="https://custom-elements-everywhere.com/">many more</a>. Additionally, TypeScript
```

Comment:
@karlbaumhauer  Providing an external link here to "custom-elements-everywhere.com" under the linktext "many more" can be ambiguous.
The linktext should describe the target, e.g.: "including React, Vu, and Angular through the use of _custom elements_."

Reply:
@miguelxnxn the point here is, that it actually inlcudes **many more** frameworks than just React, Vue and Angular. Thats why I added this.

Reply:
Could we then say:
"including React, Vu, Angular and many more through the use of _custom elements_."? This way, the link text informs about its target.

Reply:
updated